### PR TITLE
Emphasise mmCIF over PDB format in the tutorial

### PR DESCRIPTION
**Pull request by deepaklorkhatri007**
This pull request addresses issue #1102 

- [X ] I hereby agree to dual licence this and any previous contributions under both
the _Biopython License Agreement_ **AND** the _BSD 3-Clause License_.

- [X ] I have read the ``CONTRIBUTING.rst`` file, have run ``flake8`` locally, and
understand that AppVeyor and TravisCI will be used to confirm the Biopython unit
tests and style checks pass with these changes.

- [x] I have added my name to the alphabetical contributor's listings in the files
``NEWS.rst`` and ``CONTRIB.rst`` as part of this pull request, am listed
already, or do not wish to be listed. (*This acknowledgement is optional.*)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -65,6 +65,7 @@ Doc/examples/tree1.nwk
 
 #Ignore LaTeX temp files, and compiled output
 Doc/*.aux
+Doc/*.gz
```
Comment (peterjc) [inline .gitignore]: Do you have ``Doc/*.gz`` files on your machine? I wonder why...

File: .gitignore [hunk 2]
```diff
@@ -103,3 +104,6 @@ Doc/*/*/hevea.sty
 
 #Ignore unittest cache dirctory
 .cache/
+
+#Ignore vscode directory
+.vscode
```
Comment (peterjc) [inline .gitignore]: Can you include a final line break here please?

**Comment by peterjc** [comment]
I realise that #2068 would have a merge conflict with this - and given how many bits of the Tutorial I had to update, I would like to merge that as soon as possible. Since you are working on this, would you mind looking at the PDFs on that branch? If you are using Windows testing is especially appreciated.

(We can still merge this #2102 or #2068 in either order, but either way means a little extra work - probably for me)

File: Doc/Tutorial/chapter_pdb.tex [hunk 3]
```diff
@@ -1,11 +1,77 @@
...
+Bio.PDB is a Biopython module that focuses on working with crystal structures of biological macromolecules. Among other things, Bio.PDB includes a PDBParser class that produces a Structure object, which can be used to access the atomic data in the file in a convenient manner. There is limited support for parsing the information contained in the PDB header. PDB file format is no longer being modified or extended to support new content and PDBx/mmCIF became the standard PDB archive format in 2014. All the Worldwide Protein Data Bank (wwPDB) sites uses the macromolecular Crystallographic Information File (mmCIF) data dictionaries to describe the information content of PDB entries. mmCIF uses a flexible and extensible key-value pair format for representing macromolecular structural data and imposes no limitations for the number of atoms, residues or chains that can be represented in a single PDB entry (no split entries!).
```
Comment (peterjc) [inline Doc/Tutorial/chapter_pdb.tex]: Is this the only *new* text, the rest just moving things into a better order?

File: .gitignore (hunk 1 above)
Comment (deepaklorkhatri007) [reply .gitignore]: Yeah, I do have a .gz file on my machine. I used TexMaker and when I select PDFLaTeX it creates Tutorial.synctex.gz file also.

File: .gitignore (hunk 2 above)
Comment (deepaklorkhatri007) [reply .gitignore]: sure

File: Doc/Tutorial/chapter_pdb.tex (hunk 3 above)
Comment (deepaklorkhatri007) [reply Doc/Tutorial/chapter_pdb.tex]: that is correct, if you think I should add some more information please let me know :)

**Comment by deepaklorkhatri007** [comment]
> Since you are working on this, would you mind looking at the PDFs on that branch? If you are using Windows testing is especially appreciated.

I can try resolving the conflicts, I am using Fedora Linux as my main machine But I can do testing on windows if it is necessary. Can you please tell me what setup do you use for editing LaTeX I use Texmaker.

> (We can still merge this #2102 or #2068 in either order, but either way means a little extra work - probably for me)

no worries I will do my best.

File: .gitignore (hunk 1 above)
Comment (peterjc) [reply .gitignore]: OK, this addition makes sense then 👍

**Comment by peterjc** [comment]
I've added a note on #2068 about testing that. It looks like I'll have to update that branch anyway, so it can wait until your mmCIF/PDB changes are done.

File: CONTRIB.rst [hunk 4]
```diff
@@ -84,6 +84,7 @@ please open an issue on GitHub or mention it on the mailing list.
 - David Weisman <david.weisman at domain acm.org>
 - David Winter <https://github.com/dwinter>
 - Devang Thakkar <https://github.com/devangthakkar>
+- Deepak Khatri <https://github.com/deepaklorkhatri007>
```
Comment (peterjc) [inline CONTRIB.rst]: Can you move name before Devang please (simple sorting - this failed the automated check - see the comment at the top of the file).

Comment (deepaklorkhatri007) [reply CONTRIB.rst]: Really sorry about that, I have tested the Tutorial.pdf files and created a table on #2068 please take a look.

Comment (peterjc) [reply CONTRIB.rst]: Thank you - for both points.

Comment (deepaklorkhatri007) [reply CONTRIB.rst]: @peterjc please don't say thank you, it's a dream come true really, I know python and I want to contribute in python code too. is there anything I should keep in mind for that?

Comment (peterjc) [reply CONTRIB.rst]: Maybe have a browse of the open issues and pull requests, and see if anything sparks your interest?

I would also suggest installing flake8 and the pre-commit hook (see https://github.com/biopython/biopython/blob/master/CONTRIBUTING.rst for details) if you start working on our Python files.

**Comment by peterjc** [comment]
Changes applied, thank you.

File: CONTRIB.rst (hunk 4 above)
Comment (deepaklorkhatri007) [reply CONTRIB.rst]: @peterjc got it :100:   :)

**Pull request merged**